### PR TITLE
OCE-456 adding new feature

### PR DESCRIPTION
**Pull request by RahulVarma23**

### CODE REVIEW

File: src/test/java/GetCallTest.java [hunk 1]
```diff
@@ -41,6 +38,7 @@ public void testGetRequest() {
             map.put(fnames.get(i), lnames.get(i));
         }
         System.out.println(map);
+        System.out.println("---finish test");
```
Comment (LaxmanRDec19) [inline src/test/java/GetCallTest.java]: remove this line

File: src/test/java/SampleTest.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import model.Employee;
+import org.testng.annotations.Test;
+
+import static io.restassured.RestAssured.given;
+
+public class SampleTest {
+
+    @Test
+    public void testPatchRequest() {
+        String BASE_URI = "https://reqres.in/api/users";
```
Comment (sumanmishra9192) [inline src/test/java/SampleTest.java]: Make it as a instance variable.

File: src/test/java/GetCallTest.java (hunk 1 above)
Comment (mayurrm) [reply src/test/java/GetCallTest.java]: add new line

Comment (RahulVarma23) [reply src/test/java/GetCallTest.java]: done.

File: src/test/java/SampleTest.java (hunk 2 above)
Comment (RahulVarma23) [reply src/test/java/SampleTest.java]: done .thank you the suggestion!

**Review by LaxmanRDec19 — APPROVED** [review]
LGTM

**Review by sumanmishra9192 — APPROVED** [review]
LGTM

**Comment by mayurrm** [comment]
LGTM

**Pull request merged**